pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -1,4 +1,6 @@
 { runCommand, nettools, bc, perl, gmp, libmpc, mpfr, kmod, openssl
+, libelf ? null
+, eject ? null
```

Comment:
`utillinux`, not `eject`.

Otherwise looks good.

Reply:
Yeah, makes sense. Building the new version.